flakey test test_chord.test_add_chord_to_chord

```
―――――――――――――――――――――― test_chord.test_add_chord_to_chord ――――――――――――――――――――――
self = <t.integration.test_canvas.test_chord object at 0x7fd694409220>
manager = <celery.contrib.testing.manager.Manager object at 0x7fd6944093a0>
    @pytest.mark.flaky(reruns=5, reruns_delay=1, cause=is_retryable_exception)
    def test_add_chord_to_chord(self, manager):
        if not manager.app.conf.result_backend.startswith('redis'):
            raise pytest.skip('Requires redis result backend.')

        c = group([add_chord_to_chord.s([1, 2, 3], 4)]) | identity.s()
        res = c()
>       assert res.get() == [0, 5 + 6 + 7]
E       assert [18, 0] == [0, 18]
E         At index 0 diff: 18 != 0
E         Full diff:
E         - [18, 0]
E         + [0, 18]
t/integration/test_canvas.py:630: AssertionError
------------------------------ Captured log call -------------------------------
```

seems some ordering issue?

--- a/t/integration/test_canvas.py
+++ b/t/integration/test_canvas.py
@@ -630,7 +630,7 @@ class test_chord:
 
         c = group([add_chord_to_chord.s([1, 2, 3], 4)]) | identity.s()
         res = c()
-        assert res.get() == [0, 5 + 6 + 7]
+        assert sorted(res.get()) == [0, 5 + 6 + 7]
 
     @flaky
     def test_eager_chord_inside_task(self, manager):

